tests/test_utils.py

```diff
@@ -0,0 +1,156 @@
+"""Tests utility functions."""
+
+import os
+import pandas as pd
+from pandas.testing import assert_frame_equal
+import sys
+import subprocess
+import unittest
+
+if os.environ.get("DATABRICKS_RUNTIME_VERSION", None):
+    sys.path.append("../src")
+    # If your cluster is 15.0.0 or higher, you should install
+    # requirements-test.txt at the cluster level by uploading
+    # it. This approach is for older clusters.
+    # db_version = os.environ.get("DATABRICKS_RUNTIME_VERSION")
+    # major = int(db_version.split(".")[0])
+    # if major < 15:
+    #     subprocess.call(["/bin/sh", "../sh/install-requirements.sh", "test"])
+
+    import util  # noqa
```

Comment:
I think these two lines can be outdented. They should be included no matter where the tests are running. And then we don't need the `# noqa`.

Reply:
Also, in the rare cases where we do need `# noqa` it should ideally be narrowed to only the exact check ID that is the issue, e.g. `# noqa F401` or whatever.

Reply:
hmm but utils imports `from databricks.sdk.runtime import *` which won't work in github right? So doesn't it have to be indented? 

Ah okay I can track down the id

Reply:
i guess nevermind i removed it and the linter had no problem?